prog/vm/github_runner.rb

```diff
@@ -295,12 +295,12 @@ def setup_info
       #
       # Hence, the condition is added to check if the workflow_job is nil or
       # the conclusion is failure.
-      if !github_runner.workflow_job || github_runner.workflow_job.fetch("conclusion", "failure") == "failure"
+      if (job = github_runner.workflow_job).nil? || job.fetch("conclusion", "failure") == "failure"
```

Comment:
Now, I must confess, the assignment wrapped in a logical expression is a bit 
![image](https://github.com/ubicloud/ubicloud/assets/7191/23bf7692-ea0b-4790-8aa3-b984037d5056)